src/main/java/core/basesyntax/CalculateAble.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface CalculateAble {
```

Comment:
It needs to be divided into two interfaces. One Drawable with a draw() method to display the shape. The second AreaCalculator with the getArea() method to calculate the area.